### DISCUSSION TITLE
Idea to make LBB really portable to be used in a vehicle.

### DISCUSSION THREAD

**Discussion by munecito** [outdoorbits/little-backup-box#148] [Ideas]
I have ordered a couple of buck converters 12-24Vdc to 5Vdc 5a.

They are only small so my idea is to connect them straight to a 12V source from the car so the backups can be done while driving after a day's work.

The reasons why I am thinking of implementing this are mainly three:

-When I do jobs away sometimes I have two to three hours drive after a job and sometimes I have arrived to a hotel quite late, which means getting to bed really late because of the need to do the backups.

-I would like to get a reliable power supply in the car. I can use a cigarette plug charger, but sometimes they become lose so the pi could turn off during the backup and there's a risk of corrupting the backup or even the pi. So I'm thinking about adding a dedicated Anderson plug to supply the power.

-The USB 3 ports have been reported to be underpowered which causes some drives to disconnect. Supplying constant and reliable voltage and current to an active card reader would potentially resolve these disconnections.

I'm waiting for the parts to arrive so I can test. 

If it works. What would be the chances that I could send a couple of these buck converters to Kerry Staite to see if they could be integrated somehow in the 3d model of the box?

If that works the whole LBB and USB hub could be ran from a single 12v power supply. The box would be marginally bigger, but the disconnection issue would be gone. 

Opinions? Ideas?

**Comment by outdoorbits** [comment]
Sounds interesting. For another project I use a LM 2596 T5,0 which converts a wide range of input voltages to 5V. At least this device gets very hot. If something like this would be integrated inside the case, active cooling could be to consider.
I also tried to contact Kerry Staite months ago to ask him for some minor modifications about the display but didn't get any answer yet.
Maybe best way would be to play around with something like freecad and design a new case. I would appreciate to publish the CAD-files, not the stl-files only. I did some constructions using freecad for 3D-printing. It's not a rocket-science but my tries to construct a case failed in lack of time.
But maybe an external solution would be even easier. What's about using a powerful power bank? You can charge it overnight and using something like my 2700mAh device you can do really large backups while driving home.

**Reply by munecito** [reply]
I spoke via email with Kerry. While he's a great guy and was very gracious with the time we spent bouncing ideas, I think he is busy with other projects so he can't really dedicate any time to modify the box.

He told me that he had signed up the box project to thingyverse.  The files can be pulled from there to remix them.  I have remixed a couple of things in the past so I may try to do what is needed for my mod.

**Comment by black-jmyntrn** [comment]
I'm working on this now, only thing is I am using a couple 18650 batteries and a USB-C Passthrough Charger board. It will be used mainly with me in my car or close to it. The problem now is I want to change the LBB IP as the Streamdeck running through companion pi uses 8000 as well and I think I'll need to use a usb hub that's powered as every time I insert the USB cad reader I hear the HDD sound like it restarts... umm ill actually try leaving it installed and only removing the memory card now that I think while typing all of this

**Comment by black-jmyntrn** [comment]
you know... if LBB could run on a router board, that would be EPIC!
I have a banana Pi R3 and will be ordering a Banana Pi R4 and if I could use the R3 since it has sd card slot with a 2TB or 4TB NVME drive... I'm sure it would be plenty fast. Too bad the R3 Mini doesn't have sd card slot or that lil bugger would be pocket sized!

**Comment by outdoorbits** [comment]
There are some functions of lbb (display and buttons) depending on other projects and they are bound to Raspberry pi. If you don't need them, you could remove them more or less easily. Another problem would be that (if I'm right) there is no debian based linux for banana pi. So package handling would be different and the packages might not be the same. This makes it a big effort to adapt lbb to both environments. What would be the benefit?
